model/godleyIcon.h

```diff
@@ -111,7 +125,14 @@ namespace minsky
     const Variables& stockVars() const {return m_stockVars;}
     GodleyTable table;
     /// updates the variable lists with the Godley table
-    void update();
+    void update() override;
```

Comment:
Obviously, when you remove the virtual base class method, you'll need to remove the override.

Reply:
Kept it for the above reasons.